spec/models/form/setup/questions/created_by_id_spec.rb

```diff
@@ -20,6 +20,10 @@
     }
   end
 
+  before do
```

Comment:
Is this just to avoid hitting the database?

Reply:
I have validations for name presence, can't create one without it - so have to avoid using DB, but presumably this test is still valid as we potentially have users on staging/prod maybe that don't have names

Reply:
That might be worth a comment in the code since it's not obvious

Reply:
I don't think we should test a case where the user is in an invalid state, it would never be saved as such.

Reply:
Is that validation correct? I thought the reason we did that originally was that we can't guarantee the value is populated in the legacy system? If so what do we do for import?